eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Tools.hs

```diff
@@ -129,24 +123,26 @@ evaluateTransactionExecutionUnits pp tx utxo ei sysS costModels =
   let getInfo :: Language -> Either (TranslationError (Crypto era)) (Language, VersionedTxInfo)
       getInfo lang = (,) lang <$> txInfo pp lang ei sysS utxo tx
    in do
-        ctx <- mapM getInfo (Set.toList $ languagesUsed (Map.elems scripts))
+        ctx <- mapM getInfo languagesUsed
         pure $
           Map.mapWithKey
-            (findAndCount pp (array (PlutusV1, PlutusV2) ctx))
+            (findAndCount pp (array (minBound :: Language, maxBound) ctx))
```

Comment:
I like that. I thought about suggesting it too ;)

Reply:
This actually won't work, because if `ctx` is missing a `Language` this will be bottom.